### PR TITLE
Feat/new component

### PR DESCRIPTION
**Pull request by leungnamson1023**
https://trello.com/c/BEsNF39g/28-test-about-component-to-make-sure-props-are-passed-down-small

### CODE REVIEW

File: spec/SideBar.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow } from 'enzyme';
+
+import SideBar from '../client/src/SideBar.jsx';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (tmkuba) [inline spec/SideBar.test.js:8]: If you end up with a lot of components, you can create a `setupTests.js` file and configure jest to load it before each test so you don't have to repeat the Enzyme.configure in every file.  With a few components, it's not really a big deal though:
https://github.com/airbnb/enzyme/blob/master/docs/guides/jest.md

File: spec/SideBar.test.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+describe('About Component', () => {
+  test('should render correctly', () => {
+    expect(shallow(<SideBar />)).toMatchSnapshot();
+  });
+});
+
+describe("rendered `About`", () => {
+  it("received two props", () => {
```
Comment (tmkuba) [inline spec/SideBar.test.js]: test description doesn't match the expect (which is looking for 4 props)

**Review by tmkuba — CHANGES_REQUESTED** [review]
Looks good Namson, one small change request on the test description.  The other one re: Enzyme setup isn't a blocker

File: spec/SideBar.test.js (hunk 1 above)
Comment (leungnamson1023) [reply spec/SideBar.test.js:8]: Got it!

File: spec/SideBar.test.js (hunk 2 above)
Comment (leungnamson1023) [reply spec/SideBar.test.js]: updated test

**Pull request merged**